### PR TITLE
[9.0][hr_timesheet_activity_begin_end] migration to v9

### PR DESCRIPTION
**Pull request by Martronic-SA**
model hr.analytic.timesheet has been removed so we have moved functions to account.analytic.line and changed model of related views.
Same as https://github.com/OCA/timesheet/pull/96
But with right author.

### CODE REVIEW

**Comment by Martronic-SA** [comment]
@alexey-pelykh @leemannd @robinkeunen @houssine78 
Sorry for the mess but that PR was very old and had been made with wrong author.

**Comment by vvrossem** [comment]
@Martronic-SA thank you :-)

File: hr_timesheet_activity_begin_end/__openerp__.py [hunk 1]
```diff
@@ -1,36 +1,22 @@
...
- 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'version': '9.0.1.0.0',
+ 'author': 'Camptocamp,Martronic SA,Odoo Community Association (OCA)',
  'license': 'AGPL-3',
  'category': 'Human Resources',
- 'depends': ['hr_timesheet_sheet',
-             ],
+ 'depends': [
+     'hr_timesheet_sheet',
+     'float_time_hms',
```
Comment (vvrossem) [inline hr_timesheet_activity_begin_end/__openerp__.py]: @Martronic-SA where could we find the `float_time_hms` module ?

Comment (Martronic-SA) [reply hr_timesheet_activity_begin_end/__openerp__.py]: Sorry this was our own version of the module with float_time_hms (means Hour Minute Second).
You can find the module here:
https://github.com/jordillinares/addons
As long as we are using the activity begin end module in telephonic  hotline, we wanted to have something more accurate than minutes.

**Comment by houssine78** [comment]
@alexey-pelykh I think that your change request has been taken into account ! Thank you

**Comment by alexey-pelykh** [comment]
But builds seems to be failing

**Comment by houssine78** [comment]
Yes I saw this. Not sure is related to this PR though

image

**Comment by alexey-pelykh** [comment]
We'll have to wait for that to be green, unfortunately :(

File: hr_timesheet_activity_begin_end/views/hr_timesheet_sheet.xml [hunk 2]
```diff
@@ -7,12 +7,12 @@
       <field name="inherit_id" ref="hr_timesheet_sheet.hr_timesheet_sheet_form"/>
       <field name="arch" type="xml">
         <xpath expr="//field[@name='timesheet_ids']/tree/field[@name='unit_amount']" position="before">
-          <field name="time_start" widget="float_time"/>
-          <field name="time_stop" widget="float_time"/>
+          <field name="time_start" widget="float_time_hms"/>
```
Comment (vvrossem) [inline hr_timesheet_activity_begin_end/views/hr_timesheet_sheet.xml]: @Martronic-SA while testing again the module, we encountered an error : `float_time_hs` should be removed from this file as well

**Pull request closed without merging**